### DISCUSSION TITLE
Step-by-step guide to sending Nightingale alerts to Feishu

### DISCUSSION THREAD

**Discussion by xiangzilong** [ccfos/nightingale#1227] [Ideas]
夜莺接入飞书告警保姆教程
https://blog.csdn.net/qq_36270681/article/details/127517997

**Comment by UlricQin** [comment]
<img width="880" alt="image" src="https://user-images.githubusercontent.com/2769466/197920352-1697e8d1-596c-4bed-b576-222ed54473a7.png">
